README.md

```diff
@@ -55,9 +55,22 @@ HARRY HALPIN Research Scientist at Massachusetts Institute of Technology [↩](#
 ### Challenges KYODO meets
 
 - Create a framework for liquid/sustainable/ horizontal organizations, that could serve as a tool to manage its activity, be incentive and efficient. 
-- Test the new reward mode for #DAO members activity, that matches the post labor era features (fun/work, no long-term obligations, follow self-interests)
+- Test the new reward model for #DAO members activity, that matches the post labor era features (fun/work, no long-term obligations, follow self-interests)
 -  Achieve a network effect that brings mass adoption closer.
 
+## What can KYODO do?
+
+As soon as KYODO Dapp created to manage existing DAO Decent.Fund, this tool solves the real issues DF DAO met on its way. DF DAO needs are common to others DAO: many active members; self-motivation; hyper-flexibility to changing members and switching tasks; stable currency to reward contributors; effective fund management; good UX to eliminate the entry barrier for newcomers; ability to scale. We try different models and came up with the next list of features, that are already going through reality check:
+
+* Creating a working group (DAO), KYODO provides a clear interface for management processes, also the constructor to make a customized version of a DAO.
+* Mint your own token or use DF token (Decent.fund token - KYODO creator) feature based on Colony
```

Comment:
Create you own token or use existing one.
If you are creating a new Kyodo, new Colony will be created for you, newly created Colony is the owner of the token, we cannot pass ownership of our token to external colonies